common/fs.h

```diff
@@ -58,10 +58,11 @@ class FSList : public Array<FSNode> {};
 class FSNode : public ArchiveMember {
 private:
 	SharedPtr<AbstractFSNode>	_realNode;
-	FSNode(AbstractFSNode *realNode);
 
 public:
-	/**
+    FSNode(AbstractFSNode *realNode);
```

Comment:
Actually, since this is public now, we need to specify that FSNode takes the ownership of the pointer and will delete it at some point. Please add a comment for that.